examples/apache/matrix-synapse.conf

```diff
@@ -50,6 +51,15 @@
 		Header always set Content-Type "application/json"
 		Header always set Access-Control-Allow-Origin "*"
 	</Location>
+	# Map /.well-known/matrix/server for server discovery
+	Alias /.well-known/matrix/server /matrix/static-files/.well-known/matrix/server
+	<Files "/matrix/static-files/.well-known/matrix/server">
+		Require all granted
+	</Files>
+	<Location "/.well-known/matrix/server>
+		Header always set Content-Type "application/json"
+		Header always set Access-Control-Allow-Origin "*"
```

Comment:
`Header always set Access-Control-Allow-Origin "*"` is not necessary for the `server` file, as it's not meant to be used by browsers and CORS rules don't apply.